### ISSUE TITLE
Fix On-Street Parking Data Collector

### ISSUE THREAD

**Issue by rcavaliere** [noi-techpark/bdp-commons#664]

**Comment by dulvui** [comment]
I tested now the push endpoint and it works fine. I send the test data that is available in the repository and it worked.
The endpoint I used is https://parking.opendatahub.bz.it/dc-onstreet-parking-bz/json/pushRecords

Could it be that there are problems on data provider site?

**Comment by rcavaliere** [comment]
@dulvui yes it could be, this is the feedback I wanted to get. I will contact the Data Provider and let you know.

**Comment by rcavaliere** [comment]
News here: the tech provider here (Axians) is changing platform and will push data back soon. Probably we will need to change something (little things) in our Data Collector

**Comment by rcavaliere** [comment]
@clezag we have some news here. The new end-point is mqtts://xsona-broker.axians.it:8443, we must subscription to the topic application /6f599389-c6b0-4a76-843d-8d493192c4c0/device/+/event/up. I provide you the credentials separately. Please note that the sensors are still not connected to this back-end, we should now just test if the connection is working

**Comment by clezag** [comment]
@rcavaliere this is the "parking-onstreet-bz" data collector (origin=AXIANS), right? 
As far as I understand, it currently exposes a REST endpoint that they used to push data, but now we need to connect to a MQTT broker instead?
Just making sure I'm not looking at the wrong thing

**Comment by rcavaliere** [comment]
@clezag yes, correct. No, it will work in the past, with them pushing us the data. Just check now if the connection is OK.

**Comment by clezag** [comment]
@rcavaliere I've tried connecting with two tools, but I get timeouts:

If you have docker installed you can try, too:
```sh
docker run hivemq/mqtt-cli subscribe --topic '/6f599389-c6b0-4a76-843d-8d493192c4c0/device/+/event/up' --host xsona-broker.axians.it --port 8443 --identifier 'noitest' --user '***' --password '****' --secure -v
```

**Comment by rcavaliere** [comment]
@clezag thanks for the check, I have notified the issue to the Data Provider. Let's put the task in stand-by until we have a feedback.

**Comment by rcavaliere** [comment]
@clezag the technological provider of the platform asks to have a troubleshooting session in order to more quickly solve the issue. Can you manage this? If yes, I will put you in contact with the reference software developer.

**Comment by clezag** [comment]
@rcavaliere sure, just send me their contact

**Comment by rcavaliere** [comment]
@clezag just shared via e-mail the reference contacts of Axians

**Comment by clezag** [comment]
@rcavaliere We've just had a call and end the end we were just using the wrong port. Correct is `8883` and not `8443`. 

Now I'll wait and check if we get some data and how it looks
